analysis/mailchimp/mailchimp_gains_losses.sql

```diff
@@ -48,14 +48,25 @@ months as (
 	union
 	select month
 	from losses
+),
+
+monthly_gains_and_losses as (
+
+  select
+  	months.month, nvl(total_gains,0) as month_gains,
+  	nvl(total_losses,0) as month_losses
```

Comment:
@jthandy `-nlv(...)` Might as well just negate losses here to make everything more compact below.